myft-digest-promo/main.scss

```diff
@@ -0,0 +1,100 @@
+$icon-width-s: 90px;
+$icon-width-m: 90px;
+
+.n-myft-digest-promo {
+  @include clearfix();
+  @include oGridColspan($span: 12);
+
+  display: none;
+  position: relative;
+  float: none;
+  background-color: #fdf8f2;
+  border-bottom: 1px solid #cec6b9;
+  border-top: 1px solid #cec6b9;
```

Comment:
Rather than setting hash codes, we want to be pulling colours with `getColor` or `getColorFor` if any of them fall into the use cases (I haven't looked into whether they do or not, sorry 😶): https://github.com/Financial-Times/n-ui/tree/master/colors

Here's all the origami colours we have defined: http://registry.origami.ft.com/components/o-colors@3.5.2

If the colours we need aren't in there then, uh, I guess, try and check with a designer that we can switch it to a similar origami colour? Or if there's justification enough maybe we can add a new one to n-ui (this might need more discussion).